src/web/frontend/components/Lobby/Table.tsx

```diff
@@ -0,0 +1,183 @@
+import React from "react"
+import {Row, roomArr} from './Row'
+
+interface myState {
+    array: roomArr[];
+    instantList: roomArr[];
+}
```

Comment:
It hurts my soul to see variable names like this `array`, `myState`.
Also `instantList` variable seems redundant to me since you already have all data in `array`.